trade-finance-manager-ui/templates/utilisation-reports/add-to-an-existing-payment.njk

```diff
@@ -22,7 +24,12 @@
       reportedFeesHeader: 'Reported fee'
     }
   }) }}
-  {# TODO FN-1727: Display available payments. #}
+
+  {{ paymentGroupRadioInput.render({
```

Comment:
Pedantic but could be highlight this is a radio group rather than a single radio from a group in the macro name, e.g. `paymentGroupRadioInputGroup`